ui-admin/src/portal/publish/PortalEnvDiffView.tsx

```diff
@@ -124,6 +109,8 @@ export default function PortalEnvDiffView(
     { userHasPermission(user, portal.id, 'publish') &&
       <span>Select changes to publish</span>
     }
+    <Link to={studyDiffPath(portal.shortcode, studyShortcode!, destEnvName, sourceEnvName)}
+      className="ms-3">reverse diff</Link>
```

Comment:
I wonder if we should even advertise doing this, just because it's getting into real "power-user" stuff. I could see this being here for superusers when things blow up and they ask for help from us. But I'm not sure, I don't really know when you'd need to use this anyway.

Reply:
yeah, and I wouldn't expect them to necessarily know what "reverse diff" means.  I'll leave it for now, and we can see what Rachel and friends think